### PR TITLE
Saimon/update servers

### PR DESCRIPTION
**Pull request by saimonation**

### CODE REVIEW

File: cterasdk/core/servers.py [hunk 1]
```diff
@@ -10,6 +13,27 @@ class Servers(BaseCommand):
...
+
+        :param str name: Name of the server
+        :param list[str] include: List of fields to retrieve, defaults to ['name']
+        :return: The server, including the requested fields
+        """
+        include = union(include or [], Servers.default)
+        include = ['/' + attr for attr in include]
+        server = self._portal.get_multi('/servers/%s' % name, include)
+        if server.name is None:
+            raise CTERAException('Could not find server', None, name=name)
```
Comment (ygalblum) [inline cterasdk/core/servers.py:34]: In other places the caller can distinguish between the type of errors. For example, to check for 404 (this pattern is used a lot in the Ansible modules).  We should raise something that can be checked programmatically

Comment (saimonation) [reply cterasdk/core/servers.py:34]: @ygalblum shouldn't we incorporate this across the board then? add a "ObjectNotFound" exception that gets thrown when we can't find an object. Separate task/branch?

Comment (ygalblum) [reply cterasdk/core/servers.py:34]: When the API implementation calls "get" on an object that does not exist, the exception the user receives already includes the 404 code. The problem occurs only when the API implementation calls "get_multi". So, not sure we need to translate all failures to a new exception. Maybe just mimic the 404 exception in cases where get_multi is used and the object was not found

Comment (saimonation) [reply cterasdk/core/servers.py]: Fixed

**Review by ygalblum — APPROVED** [review]

**Pull request merged**